changelog.d/10426.feature

```diff
@@ -0,0 +1 @@
+Email notifications now state whether an invitation is to a room or a space.
```

Comment:
Not sure this is really "feature"-worthy?

Reply:
I'd say it is since it's a user-facing change (that's not a bugfix)